README.md

```diff
@@ -294,6 +290,18 @@ Options passed to `start_link/2` as the options `{client_opts, [{connection_opts
   When a timeout happens, the connection is closed and the client attempts to
   set up a new connection. See the client option `node_down_timeout` above.
 
+#### Deprecated connection options
+
+* `{tcp_connect_timeout, timeout()}`
+
+  Timeout passed to `gen_tcp:connect/4`. Default infinity.
+  Replaced by `{connect_timeout, timeout()}`.
```

Comment:
I don't think we need a separate heading for these ones. There are already so many levels it gets hard to keep track.

Let's just list them as options and write Deprecated after each of them?

```suggestion
* `{tcp_connect_timeout, timeout()}`

  Timeout passed to `gen_tcp:connect/4`. Default infinity.
  Deprecated. Replaced by `{connect_timeout, timeout()}`.
```